src/index.js

```diff
@@ -0,0 +1,84 @@
+const crypto = require('crypto');
+const assert = require('assert');
+
+const key = process.env.HASHING_KEY;
+assert(key, 'HASHING_KEY not provided');
+
+const ACTION = {
+  ENCRYPT: 'ENCRYPT',
+  DECRYPT: 'DECRYPT'
+};
+
+/**
+   * Encrypt/Decrypt incoming data depending on action
+   * @param  {Array|string|boolean|object} data - data to encrypt/decrypt.
+   * In case of encryption, data can be Array of string or string.
+   * In case of decryption, data can be an Array of objects or an object
+   * @param {string} action - action to perform. Can be either ENCRYPT or DECRYPT
```

Comment:
Do you mean something like
```
const actionEnum = {
  ENCRYPT: 0,
  DECRYPT: 1
};
```

I don't get the point of doing something like that, I tried to document myself but I didn't find any relevant informations